projects/igniteui-angular/src/lib/carousel/carousel.component.ts

```diff
@@ -77,17 +79,18 @@ export class IgxCarouselComponent implements OnDestroy {
 
     /**
      * Sets whether the carousel can pause the slide transitions.
-     *
+     * This regards to user interactions only.
```

Comment:
Just make it sound more natural:  Sets whether the carousel will pause the slide transitions on user interactions.